src/stores/widgets/StopGapWidget.ts

```diff
@@ -415,7 +415,7 @@ export class StopGapWidget extends EventEmitter {
     private feedEvent(ev: MatrixEvent) {
         if (!this.messaging) return;
 
-        const raw = ev.event as IEvent;
+        const raw = ev.getClearEvent() as IEvent;
```

Comment:
Hmm, will this not be unexpected given a `clearEvent` only has a type & content, is missing all the other fields

Reply:
https://github.com/matrix-org/matrix-js-sdk/pull/1784/commits/326b14402f5cc613bd4f2a8e6d011e6dbff6aa29